src/cpp/thread_manager/thread_manager.h

```diff
@@ -124,13 +124,15 @@ class ThreadManager {
     WorkerThread(ThreadManager* thd_mgr);
     ~WorkerThread();
 
+    bool isCreated();
```

Comment:
Let's rename this to `created()` since it's a simple getter. And also label it `const` (and just place the definition inline).

Reply:
Renamed isCreated and labeled it const, as suggested.